arcgis-ios-sdk-samples/Maps/Show location history/README.metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+    "category": "Maps",
+    "description": "Display your location history on the map.",
+    "ignore": false,
+    "images": [
+        "image1.png"
+    ],
+    "keywords": [
+		"GPS", 
+		"bread crumb", 
+		"breadcrumb", 
+		"history", 
+		"movement", 
+		"navigation", 
+		"real-time", 
+		"trace", 
+		"track", 
+		"trail"
+    ],
+    "redirect_from": [
+    ],
+    "relevant_apis": [
+		"AGSCLLocationDataSource",
+		"AGSGraphic",
+		"AGSGraphicsOverlay",
+		"AGSLocationDataSource",
+		"AGSLocationDisplay",
+		"AGSMap",
+		"AGSMapView",
+		"AGSPolylineBuilder",
+		"AGSSimpleLineSymbol",
+		"AGSSimpleMarkerSymbol",
+		"AGSSimpleRenderer"
+    ],
+    "snippets": [
+        "LocationHistoryViewController.swift",
+        "LocationTracking.swift"
```

Comment:
Please remove this line and the comma from the previous line.

Reply:
Fixed in [`2e108ddc`](https://github.com/Esri/arcgis-runtime-samples-ios/pull/751/commits/2e108ddc08e24501431b9b8cefae28dfdc4e5c5d).